src/SourceNodes/medianode.ts

```diff
@@ -266,7 +266,7 @@ class MediaNode extends SourceNode {
         }
 
         if (
-            this._startTime - this._currentTime <= this._preloadTime &&
+            this._startTime - this._currentTime <= this._preloadTime * this._playbackRate &&
```

Comment:
This is so that nodes will start preloading sooner/later when playback rate is different than 1.0